chart/cas-cif/templates/cronJobs/db-init.yaml

```diff
@@ -14,62 +12,35 @@ spec:
       activeDeadlineSeconds: 600
       template:
         metadata:
-          labels:
-            name: {{ template "cas-cif.fullname" . }}-db-init
-            release: {{ .Release.Name }}
-{{ include "cas-cif.labels" . | indent 12 }}
+          labels: {{ include "cas-cif.labels" . | nindent 12 }}
+          name: {{ template "cas-cif.fullname" . }}-db-init
         spec:
           restartPolicy: Never
           imagePullSecrets: {{ include "cas-cif.imagePullSecrets" . | nindent 12 }}
           containers:
-            - env:
-                - name: PGDATABASE
-                  value: postgres
-                - name: PGUSER
-                  value: postgres
-                - name: PGPASSWORD
-                  valueFrom:
-                    secretKeyRef:
-                      key: password-superuser
-                      name: {{ template "cas-cif.fullname" . }}-patroni
-                - name: PGPORT
-                  value: "5432"
-                - name: PGHOST
-                  value: {{ template "cas-cif.fullname" . }}-patroni
+            - env: {{ include "cas-cif.superUserPgEnv" . | nindent 16 }}
                 - name: CIF_USER
                   valueFrom:
                     secretKeyRef:
-                      key: database-owner-user
-                      name: {{ template "cas-cif.fullname" . }}
-                - name: CIF_PASSWORD
-                  valueFrom:
-                    secretKeyRef:
-                      key: database-owner-password
-                      name: {{ template "cas-cif.fullname" . }}
-                - name: CIF_DATABASE
-                  valueFrom:
-                    secretKeyRef:
-                      key: database-name
-                      name: {{ template "cas-cif.fullname" . }}
+                      key: user
+                      name: {{ template "cas-cif.dbSecret" . }}
               name: {{ template "cas-cif.fullname" . }}-db-init
               image: {{ .Values.image.psql.repository }}:{{ default .Values.defaultImageTag .Values.image.psql.tag }}
               imagePullPolicy: {{ default .Values.defaultImagePullPolicy .Values.image.psql.pullPolicy }}
               resources:
                 limits:
-                  cpu: 1000m
+                  cpu: 50m
                   memory: 128Mi
                 requests:
-                  cpu: 100m
+                  cpu: 10m
```

Comment:
any risk in getting this so low?

Reply:
nah this just running psql and doesn't need any CPU